app/javascript/provincial_usages/index.js

```diff
@@ -0,0 +1,27 @@
+import * as htmlToImage from "html-to-image";
+import download from "downloadjs";
+
+OWSO.Provincial_usagesIndex = (() => {
+  function init() {
+    downloadImage();
+  }
+
+  function downloadImage() {
+    $("#downloadImage").click(function (e) {
+      e.preventDefault();
+      htmlToImage
+        .toPng(document.getElementById("table"), {
+          preferredFontFormat: "Nunito",
+          backgroundColor: "white",
+          style: { color: "black" },
+        })
+        .then(function (dataUrl) {
+          download(dataUrl, "report.png");
```

Comment:
@radinreth give a filename refers to the fact "provincial_usage_report-20210705151000.png

Reply:
noted bong